tox.ini

```diff
@@ -15,22 +15,24 @@ deps =
     sphinx16: Sphinx<1.7
     sphinx17: Sphinx<1.8
     sphinx18: Sphinx<1.9
-    sphinx21: Sphinx~=2.1.0
-    sphinx22: Sphinx~=2.2.0
-    sphinx23: Sphinx~=2.3.0
-    sphinx24: Sphinx~=2.4.0
-    sphinx30: Sphinx~=3.0.0
+    sphinx21: Sphinx<2.2
+    sphinx22: Sphinx<2.3
+    sphinx23: Sphinx<2.4
+    sphinx24: Sphinx<2.5
```

Comment:
I think we can just test the latest version of each major release, no? This seems like a lot of overkill.

Reply:
I think this goes back to what versions we officially support. This extension is installed by default, so I think we should be careful here and try to not break the builds on any version. But if we are good just supporting the latest version of each release, I'm ok with that.

Reply:
Yea, I guess for this it's good to be extra careful 👍